javascript-data-model/order-history.js

```diff
@@ -0,0 +1,85 @@
+var orderHistory = [
+  {
+    OrderNumber: '114 - 3941689 - 8772232',
+    item: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    returnWindow: 'sep, 7, 2020',
+    priceOfTheItem: 31.55,
+    FinalPrice: 34.00,
+    deliveryDate: 'august, 8, 2020',
+    orderPlacedDate: 'august, 4, 2020',
+    shipTo: 'JS Masher',
+    delivered: true,
+    eligibleForReturn: true,
+    returned: false,
+    productSupport: false,
+    productReview: null,
+    deliveryToResident: true
+  },
+  {
+    OrderNumber: '113-9984268-1280257',
+    item: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    returnWindow: 'aug, 19, 2020',
+    priceOfTheItem: 41.33,
+    FinalPrice: 44.53,
+    deliveryDate: 'July, 20, 2020',
+    orderPlacedDate: 'July, 19, 2020',
+    shipTo: 'JS Masher',
+    delivered: true,
+    eligibleForReturn: true,
+    returned: false,
+    productSupport: false,
+    productReview: null,
+    deliveryToResident: true
+  },
+  {
+    OrderNumber: '114-2875557-9059409',
+    item: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for Wii U, PC.Support Turbo and Vibration Features.No Driver and No Lag- Gamecube Adapter',
+    author: null,
+    returnWindow: 'aug, 5, 2020',
+    priceOfTheItem: 15.98,
+    FinalPrice: 17.22,
+    deliveryDate: 'July, 7, 2020',
+    orderPlacedDate: 'July, 4, 2020',
+    shipTo: 'JS Masher',
+    delivered: true,
+    eligibleForReturn: false,
+    returned: false,
+    productSupport: false,
+    productReview: null,
+    deliveryToResident: true
```

Comment:
You have the data about the items for the order intermixed with the data about the order itself, which is messy. You should organize all the data about the items in the order into a property of the order object.